com.basemark.BasemarkGPU.appdata.xml

```diff
@@ -0,0 +1,129 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Ideally appdata, desktop launcher and icons should be part of tarball you're using to build flatpak from.

Reply:
Current tarball is same output as shared to non-flatpak users and would most likely keep it that way for now. However, we can take note of this and prepare internal repositories to include at-least desktop launcher and icons (but that would only affect future versions of Basemark GPU).

Reply:
Yeah, it's not a blocker by any means.